packages/replay-next/components/sources/log-point-panel/LogPointPanel.tsx

```diff
@@ -157,15 +158,21 @@ function PointPanelWithHitPoints({
   // Otherwise fall back to using the global execution point.
   const executionPoint = closestHitPoint ? closestHitPoint.point : currentExecutionPoint;
   const time = closestHitPoint ? closestHitPoint.time : currentTime;
-  const pauseId = getPauseIdSuspense(client, executionPoint, time);
-  const frames = getFramesSuspense(client, pauseId);
-  const frameId = frames?.[0]?.frameId ?? null;
   let pauseAndFrameId: PauseAndFrameId | null = null;
-  if (frameId !== null) {
-    pauseAndFrameId = {
-      frameId,
-      pauseId,
-    };
+  try {
+    const pauseId = getPauseIdSuspense(client, executionPoint, time);
+    const frames = getFramesSuspense(client, pauseId);
+    const frameId = frames?.[0]?.frameId ?? null;
+    if (frameId !== null) {
+      pauseAndFrameId = {
+        frameId,
+        pauseId,
+      };
+    }
+  } catch (errorOrPromise) {
+    if (isThennable(errorOrPromise)) {
+      throw errorOrPromise;
+    }
```

Comment:
Do we _expect_ this error to happen? If not, we should probably at least log it to the console.

If we do though, can we add an inline comment with an explanation of why?